test-cases/tests/products/h24-verify-selfmanaged-apicast-and-custom-policy.md

```diff
@@ -0,0 +1,196 @@
+---
+products:
+  - name: rhoam
+    environments:
+      - osd-fresh-install
+estimate: 2h
+tags:
+  - manual-selection
+---
+
+# H24 - Verify self-managed APIcast and custom policy
+
+## Description
+
+This test case should prove that it is possible for customers to deploy self-managed APIcast and use custom policies on it. The 3scale QE team will perform this test case in RHOAM each time there is an upgrade of 3scale. We (RHOAM QE) should only perform this if there are modifications on our end that might break the functionality - typically changes in permissions in RHOAM and/or OSD.
+Additional context can be found in [MGDAPI-370](https://issues.redhat.com/browse/MGDAPI-370)
+
+## Steps
+
+_1. This test case must be done using `customer-admin` user (or any other user from `dedicated-admin` group). Do not use `kubeadmin`!_
+_2. Create a new namespace (e.g. selfmanaged-apicast) for self-managed APIcast_
+
+- `oc project selfmanaged-apicast`
+- be sure to use this project for the `oc` commands
+
+_3. Create a secret with credentials to `registry.redhat.io`_
+
+- `oc create secret docker-registry redhatio --docker-server=registry.redhat.io --docker-username=<username> --docker-password=HIDDEN --docker-email=<email>`
+
+_4. Configure the pull secret_
+
+- `oc secrets link default redhatio --for=pull`
+
+_5. Import the image_
+
+- `oc import-image 3scale-amp2/apicast-gateway-rhel8:3scale<version> --from=registry.redhat.io/3scale-amp2/apicast-gateway-rhel8:3scale<version> --confirm`
+- go to [catalog](https://catalog.redhat.com/software/containers/search), find the image and see the available tags (version)
+
+_6. Clone the Example custom policy_
+
+- `git clone git@github.com:3scale/apicast-example-policy.git`
+- `cd apicast-example-policy`
+
+_7. Edit the `openshift.yaml`_
+
+Replace
+
+```
+    strategy:
+      type: Source
+      sourceStrategy:
+        from:
+          kind: ImageStreamTag
+          name: 'amp-apicast:${AMP_RELEASE}'
+```
+
+with
+
+```
+    strategy:
+      type: Source
+      sourceStrategy:
+        from:
+          kind: ImageStreamTag
+          namespace: selfmanaged-apicast
+          name: apicast-gateway-rhel8:3scale2.9.1
```

Comment:
```suggestion
          name: apicast-gateway-rhel8:3scale<version>
```
would it be also worth mentioning how to find out the 3scale version?

Reply:
apparently, this is not clear enough. that whole `3scale<version>` is an image tag. So you need to look in the catalog and see what tags are available and pick one - likely the one that corresponds with the 3scale version in RHOAM. Will improve the wording.